.gitlab-ci.yml

```
@@ -120,6 +120,26 @@ trigger:mender-dist-packages:
     - tags
     - master
 
+trigger:mender-docs-site:
+  image: alpine
+  stage: trigger
+  before_script:
+    - apk add --no-cache curl
+  script:
+    - curl -v -f -X POST
+      -F token=$MENDER_DOCS_SITE_TRIGGER_TOKEN
+      -F ref=master
+      https://gitlab.com/api/v4/projects/16842368/trigger/pipeline
+    - curl -v -f -X POST
+      -F token=$MENDER_DOCS_SITE_TRIGGER_TOKEN
+      -F ref=production
+      https://gitlab.com/api/v4/projects/16842368/trigger/pipeline
+  rules:
+    - if: '$CI_COMMIT_REF_NAME !~ /^(master|[0-9]+\.[0-9]+\.x)$/'
+      when: never
+    - changes:
+      - Documentation/*
```

It looks good. But I think it is completely equivalent to:
```suggestion
    - if: '$CI_COMMIT_REF_NAME =~ /^(master|[0-9]+\.[0-9]+\.x)$/'
      changes:
      - Documentation/*
```

which to me reads better. Not a strong opinion, though.

On top of that, as we are not catching (nor should we catch) tags, I think is better if we use `CI_COMMIT_BRANCH` instead. For consistency with practices on the template (the `mender` pipeline is old, so we can't really compare here). For this I have an stronger opinion :)